src/IBI.WZDx/Models/RoadEvents/CdsCurbZonesReference.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using IBI.WZDx.Equality;
+
+namespace IBI.WZDx.Models.RoadEvents;
+
+/// <summary>
+/// Describes specific curb zones that are impacted by a work zone via an external reference to the
+/// curb data specification's.
```

Comment:
This summary is missing "Curb API" with link (`see href`) after "specification's", see https://github.com/usdot-jpo-ode/wzdx/blob/v4.2/spec-content/objects/CdsCurbZonesReference.md